engine-rest/engine-rest-openapi/src/main/templates/paths/message/post.ftl

```diff
@@ -0,0 +1,135 @@
+                        "processVariables" : {
+                          "aVariable" : {"value" : "aNewValue", "type": "String", 
+                                          "valueInfo" : { "transient" : true }
+                                        },
+                          "anotherVariable" : {"value" : true, "type": "Boolean"}
+                        }
+                      }
+                     },
+                   "example-2": {
+                      "summary": "POST /condition",
+                      "description": "Correlate with result",
+                      "value": {
+                        "messageName" : "aMessage",
+                        "businessKey" : "aBusinessKey",
+                        "correlationKeys" : {
+                          "aVariable" : {"value" : "aValue", "type": "String"}
+                        },
+                        "processVariables" : {
+                          "aVariable" : {"value" : "aNewValue", "type": "String",
+                                          "valueInfo" : { "transient" : true }
+                                        },
+                          "anotherVariable" : {"value" : true, "type": "Boolean"}
+                        },
+                        "resultEnabled" : true
+                      }
+                     },
+                   "example-3": {
+                      "summary": "POST /condition",
+                      "description": "Correlate with result and variables",
+                      "value": {
+                        "messageName" : "aMessage",
+                        "businessKey" : "aBusinessKey",
+                        "correlationKeys" : {
+                          "aVariable" : {"value" : "aValue", "type": "String"}
+                        },
+                        "processVariables" : {
+                          "aVariable" : {"value" : "aNewValue", "type": "String",
+                                          "valueInfo" : { "transient" : true }
+                                        },
+                          "anotherVariable" : {"value" : true, "type": "Boolean"}
+                        },
+                        "resultEnabled" : true,
+                        "variablesInResultEnabled" : true
+                      }
+                     }']
+                     />
+
+  "responses" : {
+
+    <@lib.response
+        code = "200"
+        dto = "MessageCorrelationResultWithVariableDto"
+        array = true
+        desc = "Request successful. The property `resultEnabled` in the request body was `true`.
+                The `variables` property is only returned, if the property variablesInResultEnable`
+                was set to `true` in the request."
+      examples = ['"example-1": {
+                       "summary": "Status 200 Response. The property `variablesInResultEnabled` in the request body was `false` (Default).",
+                       "description": "The Response content of a status 200",
```

Comment:
I think that this part: `The property variablesInResultEnabled in the request body was false (Default).` should go in the `description` property. `summary` should be short since it's put in a `dropdown` element in the UI.

```suggestion
                       "summary": "Status 200 Response.",
                       "description": "The Response content of a status 200. The property `variablesInResultEnabled` in the request body was `false` (Default).",
```